frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -973,7 +977,13 @@ pub mod pallet {
 
 			// create the submission
 			let deposit = Self::deposit_for(&solution, size);
-			let submission = SignedSubmission { who: who.clone(), deposit, solution };
+			let reward = {
+				let call = Call::submit(solution.clone(), num_signed_submissions);
```

Comment:
Just to confirm my understanding `Call` is the pallets call enum and `Call::submit` is a function that creates `Call::Submit` variant of the enum? And the call enum and its impls are generated in the pallet macro?

Reply:
no, i think the enum variant is `Call::submit`. A cargo flag like `#[ignore_capitalization]` is used so that the enum variant has a lowercase look to it.